recipes/mdal/bld.bat

```diff
@@ -0,0 +1,22 @@
+mkdir build
+cd build
+
+cmake  ^
+	-DCMAKE_BUILD_TYPE=Rel  ^
+	-DENABLE_TESTS=OFF ^
+	-DNETCDF_PREFIX="%LIBRARY_PREFIX%" ^
+	-DHDF5_ROOT="%LIBRARY_PREFIX%" ^
+	-DGDAL_DIR="%LIBRARY_PREFIX%" ^
+	-DGDAL_LIBRARY="%LIBRARY_PREFIX%\lib\gdal_i.lib" ^
+	-DGDAL_INCLUDE_DIR="%LIBRARY_PREFIX%\include" ^
+	-DLIBXML2_LIBRARIES="%LIBRARY_PREFIX%\lib\libxml2.lib" ^
+	-DLIBXML2_INCLUDE_DIR="%LIBRARY_PREFIX%\include\libxml2" ^
+	-DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+	-DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+	..
+
```

Comment:
You need to add `if errorlevel 1 exit \b 1` after each line in this file as batch doesn't do error checking on return codes.

Reply:
Interesting. I can see what you are saying  but none of the recipes I copied had that either. I will do that now

Reply:
added in next commit